__tests__/app.test.js

```diff
@@ -79,6 +79,34 @@ describe("NC-News", () => {
         });
     });
   });
+  describe("GET /api/articles/queries", () => {
+    test("Query Topic should filter articles by the topic value specified in the query", () => {
+      return request(app)
+        .get("/api/articles?topic=mitch")
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.articles).toHaveLength(11);
+          body.articles.forEach((article) => {
+            expect(article.topic).toBe("mitch");
+          });
+        });
+    });
+    test("Query sort_by sorts the articles by any valid column", () => {
+      return request(app)
+        .get("/api/articles?sort_by=votes")
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.articles).toBeSorted();
+        });
+    });
+    test.only("Query order sorts the articles by any valid column in ASC order or DESC", () => {
+      return request(app)
+        .get("/api/articles?sort_by=votes&order=desc")
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.articles).toBeSorted({ descending: true });
+        });
+    });
```

Comment:
nice tests for each feature in isolation; 
- can you guarantee that they all properly work together?